apps/storybook/stories/components/form-control/Form-control.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { FormControl as KvibFormControl } from "@kvib/react/src/form-control/Form-control";
+import { FormLabel } from "@kvib/react/src/form-control/Form-label";
+import { FormHelperText } from "@kvib/react/src/form-control/Form-helper-text";
+import { FormErrorMessage } from "@kvib/react/src/form-control/Form-error-message";
+import { Input } from "@kvib/react/src/input/Input";
+import { Meta, StoryObj } from "@storybook/react";
+
+const meta: Meta<typeof KvibFormControl> = {
+  title: "Komponenter/Form-control",
+  component: KvibFormControl,
+  parameters: {
+    docs: {
+      story: { inline: true },
+      canvas: { sourceState: "shown" },
+    },
+  },
+  argTypes: {
+    isRequired: {
+      description: "Toggles if input should be required",
+      table: {
+        type: { summary: "boolean" },
+      },
+      control: "boolean",
+    },
+    isInvalid: {
+      description: "Toggles if input should be invalid",
+      table: {
+        type: { summary: "boolean" },
+      },
+      control: "boolean",
+    },
+    isDisabled: {
+      description: "Toggles if input should be disabled",
+      table: {
+        type: { summary: "boolean" },
+      },
+      control: "boolean",
+    },
+    isReadOnly: {
+      description: "Toggles if input should be disabled",
+      table: {
+        type: { summary: "boolean" },
+      },
+      control: "boolean",
+    },
+  },
+};
+
+export default meta;
+type FormControlStory = StoryObj<typeof KvibFormControl>;
+
+export const FormControl: FormControlStory = {
```

Comment:
Jeg savner flere stories her (se https://kartverket.atlassian.net/wiki/spaces/TB/pages/616464426/Guidelines+for+bidragsl+ypa#%E2%9C%8F%EF%B8%8F--Dokumentere-komponenten%3A-eksempel-med-Link-komponent).

Du trenger ikke nødvendighvis en story for hver prop, men i hvert fall en egen story for invalid. Dette er viktig pga testing. Se også på hvordan Elise har satt opp dokumentasjonen for Button komponentet, for å få med blått ikon og pluss-tegn i venstre-menyen, og for å få en egen visning av hver story, med tilhørende a11y-testing. (rebase til siste versjon av master for å få med a11y).